dashboard/src/test/java/com/google/cloud/tools/opensource/dashboard/DashboardTest.java

```diff
@@ -47,54 +51,57 @@
 import com.google.common.truth.Truth;
 
 public class DashboardTest {
-  
+
   private static Path outputDirectory;
   private Builder builder = new Builder();
-  
+
   @BeforeClass
-  public static void setUp() throws ArtifactDescriptorException, IOException, TemplateException {
-    outputDirectory = DashboardMain.generate();
+  public static void setUp() throws IOException {
+    outputDirectory = Files.createDirectories(Paths.get("target", "dashboard"));
```

Comment:
`dashboardMain.generate()` at `setUp` was assuming `testDashboard()` is called immediately after it. However it didn't work when I added another test `testDashboardForRepositoryException()`. So I moved `dashboardMain.generate()` into `testDashboard()`.

Reply:
You might need to move the test to another class because the setup here is `@BeforeClass` and not `@Before` (It's expensive and read-only.)

Reply:
Moved the new tests to DashboardUnavailableArtifactTest